bin/pbiviz.js

```diff
@@ -49,8 +50,8 @@ program
     .command('package', 'Package the current visual into a pbiviz file')
     .command('validate [path]', 'Validate pbiviz file for submission')
     .command('update [version]', 'Updates the api definitions and schemas in the current visual. Changes the version if specified')
-    .option('--create-cert', 'Create new localhost certificate', createCertFile)
-    .option('--install-cert', 'Install localhost certificate', openCertFile);
+    .option('--create-cert', 'Create new localhost certificate', onCreateCertFile)
```

Comment:
I think we should get rid of `--create-cert` command. Instead we could merge --create-cert and --install-cert.

If you execute `--install-cert` but there's no certificate it must be generated.

Another idea is to get rid of both options. We could check if a certificate is presented in the storage. If not we should generate it and open the installation dialog.

What do you think?